pkgs/development/python-modules/pytest-param-files/default.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, flit-core
+, pytest
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "pytest-param-files";
+  version = "0.3.4";
+
+  src = fetchFromGitHub {
+    owner = "chrisjsewell";
+    repo = pname;
+    rev = "v" + version;
+    sha256 = "sha256-Q7wWoggJN2w2a2umQHx5TsVcugqpovBEtOKruNMZQ8A=";
+  };
+
+  format = "flit";
+
+  nativeBuildInputs = [ flit-core ];
+
+  propagatedBuildInputs = [
```

`pytest` hast to go to `buildInputs`. Otherwise it's not possible to override. This is a `pytest`-specific exception.

```suggestion
  buildInputs = [
```